src/components/networked-video-player.js

```diff
@@ -37,6 +37,10 @@ AFRAME.registerComponent("networked-video-player", {
       document.body.appendChild(container);
     }
 
+    if (!NAF.connection.adapter || !NAF.connection.adapter.getMediaStream) {
+      return;
+    }
```

Comment:
This seems unnecessary. `connection.adapter` should definitely be defined at this point, since we wait for `nafConnected` above.
Also, `getMediaStream` should definitely be defined if `adapter` is. 
Also not sure how I feel about returning silently if this condition occurred, I'd rather it fail loudly, if it does ever fail.